common/components/card/_card.scss

```diff
@@ -7,6 +7,11 @@
   padding-bottom: govuk-spacing(3);
 }
 
+strong.app-card__status {
```

Comment:
Might be better to remove the element selector. [BEM](http://getbem.com/) allows you to target elements using specific patterns so we can avoid using element selectors. 

The other benefit is it means they can be applied to any element in the markup without needing to update the styles.

Reply:
In fact, we can probably remove this declaration altogether and use `app-tag app-tag--inactive` classnames to give the same appearance.